addons/im_livechat/static/tests/embed/feedback_panel.test.js

```diff
@@ -51,6 +51,8 @@ test("Close without feedback", async () => {
     triggerHotkey("Enter");
     await contains(".o-mail-Message-content", { text: "Hello World!" });
     await click("[title*='Close Chat Window']");
+    await contains(".o-livechat-closeConfirmation-overlay");
+    await click(".o-livechat-closeConfirmation-leave");
```

Comment:
Instead of targeting the BEM classname, prefer `button { text: "Leave conversation" }`